input/fsh/examples/genomic-study-examples.fsh

```diff
@@ -0,0 +1,213 @@
+Alias: $sct = http://snomed.info/sct
+Alias: $observation-category = http://terminology.hl7.org/CodeSystem/observation-category
+Alias: $sequenceontology = http://sequenceontology.org
+Alias: $loinc = http://loinc.org
+Alias: $CodeSystem-v3-ParticipationType.html = http://terminology.hl7.org/3.1.0/CodeSystem-v3-ParticipationType.html
+Alias: $v2-0203 = http://terminology.hl7.org/CodeSystem/v2-0203
+Alias: $v3-Confidentiality = http://terminology.hl7.org/CodeSystem/v3-Confidentiality
+
+Instance: lungMass
+InstanceOf: Procedure
+Usage: #example
+* meta.profile = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomic-study"
+* extension[0].url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomics-study-analysis-ext"
+* extension[=].valueReference = Reference(Procedure/lungMass-analysis1)
+* extension[+].url = "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/genomics-study-analysis-ext"
+* extension[=].valueReference = Reference(Procedure/lungMass-analysis2)
+* status = #completed
+* code = $sct#443968007 "Whole Exome Sequencing - Sequencing of entire coding region of gene (procedure)"
```

Comment:
BTW @rhdolin - QA error on this line:

Procedure.code.coding[0] (l26/c8) | error | Wrong Display Name 'Whole Exome Sequencing - Sequencing of entire coding region of gene (procedure)' for http://snomed.info/sct#443968007 - should be one of 2 choices: 'Sequencing of entire coding region of gene (procedure)' or 'Sequencing of entire coding region of gene' (for the language(s) '--') (from Tx-Server) for 'http://snomed.info/sct#443968007'
-- | -- | --

Reply:
resolved